tests/installation/ipxe_install.pm

```diff
@@ -208,7 +208,8 @@ sub run {
     if (get_var('AUTOYAST')) {
         # make sure to wait for a while befor changing the boot device again, in order to not change it too early
         sleep 120;
-        set_bootscript_hdd if get_var('IPXE_UEFI');
+        # run set_bootscript_hdd on supermicro platfrom with UEFI and IPXE boot only
+        set_bootscript_hdd if (get_var('IPXE_UEFI') && check_var('IPMI_HW', 'supermicro'));
```

Comment:
I think you misunderstand use of `set_bootscript_hdd` it just updates information on baremetal service. It doesn't  touch  machine itself (like mention in description).  Adding 'IPMI_HW' will not fix anything.

Reply:
Adding `IPMI_HW=supermicro` can make `set_bootscript_hdd` only works on Supermicro machine, or specific supermicro machine

Reply:
It is just adding not necessary variable which nobody needs.  Adding variable will not fix anything.

Reply:
supermicro machine need this variable to run set_bootscript_hdd().
DELL machine need not it.